ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.html

```diff
@@ -1,5 +1,13 @@
-<div class="container-fluid">
-    <h1 i18n>Configuration</h1>
+<div class="panel-body">
```

Comment:
Wait, it does, but not much. I didn't notice at first. Now I see some difference